pyramus/src/main/webapp/templates/matriculation/management-edit-enrollment.jsp

```diff
@@ -0,0 +1,393 @@
+            width: 300,
+            left: 8 + 300,
+            dataType : 'select',
+            editable: true,
+            paramName: 'subject',
+            options: subjectOptions
+          }, {
+            header : 'Pakollisuus',
+            width: 200,
+            left: 8 + 300 + 8 + 300,
+            dataType : 'select',
+            editable: true,
+            paramName: 'mandatority',
+            options: mandatorityOptions
+          }, {
+            header : 'Arvosana',
+            width: 200,
+            left: 8 + 300 + 8 + 300 + 8 + 200,
+            dataType : 'select',
+            editable: true,
+            paramName: 'grade',
+            options: gradeOptions
+          }, {
+            width : 22,
+            right : 8,
+            dataType : 'button',
+            imgsrc : GLOBAL_contextPath + '/gfx/edit-delete.png',
+            tooltip : 'Muokkaa',
+            onclick : function(event) {
+              finishedAttendances.deleteRow(event.row);
+            }
+          }]
+        });
+        finishedAttendances.addRows(JSON.parse(JSDATA["finishedAttendances"]));
+        document.getElementById("addFinishedTableRow").addEventListener('click', function() {
+          finishedAttendances.addRow([-1, 'SPRING2019', 'ENA', 'MANDATORY', 'IMPROBATUR', '']);
+        });
+
+        var plannedAttendances = new IxTable($('plannedAttendancesTableContainer'), {
+          id: 'plannedAttendances',
+          columns : [{
+            dataType: 'hidden',
+            paramName: 'attendanceId'
+          }, {
+            header : 'Ajankohta',
+            left: 0,
+            width: 300,
+            dataType : 'select',
+            editable: true,
+            paramName: 'term',
+            options: termOptions
+          }, {
+            header : 'Aine',
+            width: 300,
+            left: 8 + 300,
+            dataType : 'select',
+            editable: true,
+            paramName: 'subject',
+            options: subjectOptions
+          }, {
+            header : 'Pakollisuus',
+            width: 200,
+            left: 8 + 300 + 8 + 300,
+            dataType : 'select',
+            editable: true,
+            paramName: 'mandatority',
+            options: mandatorityOptions
+          }, {
+            width : 22,
+            right : 8,
+            dataType : 'button',
+            imgsrc : GLOBAL_contextPath + '/gfx/edit-delete.png',
+            tooltip : 'Muokkaa',
+            onclick : function(event) {
+              plannedAttendances.deleteRow(event.row);
+            }
+          }]
+        });
+        plannedAttendances.addRows(JSON.parse(JSDATA["plannedAttendances"]));
+        document.getElementById("addPlannedTableRow").addEventListener('click', function() {
+          plannedAttendances.addRow([-1, 'SPRING2019', 'ENA', 'MANDATORY', '']);
+        });
+      };
+    </script>
+    
+    <link rel="stylesheet" href="//unpkg.com/purecss@1.0.0/build/forms-min.css">
```

Comment:
Are these two imports necessary? If so, what are they used for?

Reply:
Removed stylesheets, they were used in prototyping.